gulpfile.js

```diff
@@ -52,19 +52,23 @@ gulp.task('apidoc', (done) => {
   }, done);
 });
 
-gulp.task('test', () => {
-  return gulp.src('./dist-server/test/node/*.js')
+gulp.task('test:index', () => {
+  return gulp.src('./dist-server/test/index.js')
     .pipe(tape({
       reporter: faucet(),
     }));
 });
 
+gulp.task('test', () => {
+  runSequence('babel','test:index');
```

Comment:
Yup, I added those in the readme.

https://github.com/sv-bootcamp/yoda/tree/master/src/test/node